Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>./../HTML CSS
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision caf1452991207b04db8a606066ef42655883bb8d)
+++ b/.gitignore	(date 1753441991950)
@@ -1,1 +1,1 @@
-./../HTML CSS
\ No newline at end of file
+./HTML CSS
\ No newline at end of file
